docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md

```diff
@@ -0,0 +1,190 @@
+- With PMK, you can have your clusters deployed on-premises, in public clouds or at the edge. In this document, we'll explain the steps on how to create an **on-premise [BareOS](https://platform9.com/docs/kubernetes/bareos-what-is-bareos) cluster** using **PMK** and connect it to **Microsoft Azure Arc**.
+
+## Prerequisites
+
+- An Ubuntu 20.04 installed physical machine or VM. For this demonstration, we would be using single node kubernetes cluster installation. This machine would also act as our management host.
+
+- The [kubectl](https://platform9.com/learn/tutorials/kubectl) executable binary for local cluster management. If you would want to have a seperate host for cluster management, follow the below steps to [install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management) on it.
+
+  ```shell
+  sudo apt-get update
+  sudo apt-get install -y apt-transport-https ca-certificates curl
+  sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+  echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+  sudo apt-get update
+  sudo apt-get install -y kubectl
+  ```
+
+  > **Note:** The *kubectl* package is installed from Kubernetes repository, hence the Google Cloud public signing key need to be downloaded to enable the repository.
+
+  > **Note: All PMK cluster nodes would have kubectl installed once cluster is deployed.**
+
+- [Install or update Azure CLI (az) to version 2.25.0 and above](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). 
+
+  Use the below command to check your current installed version.
+
+  ```shell
+  az --version
+  ```
+
+- [Helm](https://helm.sh/docs/intro/install/) version 3+ , to install the Azure Arc agents on the cluster.
+
+  ```shell
+  curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | sudo bash
+  ```
+
+- Create Azure service principal (SP)
+  To be able to complete the scenario and its related automation, Azure service principal assigned with the *Contributor* role is required.
+
+  For creating the service principal, first login to your Azure account.
+
+  ```shell
+  az login
+  ```
+
+  Login by opening the [Microsoft devicelogin page](https://microsoft.com/devicelogin) on browser and enter the code to authenticate.
+
+  An example output of successful authentication is shown below;
+
+  ![Screenshot of MSFT Device Login](./01.png)
+
+  Once you have a successful login, create the Azure service principal (SP).
+
+  ```shell
+  az ad sp create-for-rbac -n "<service-principal-name>" --role contributor
+  ```
+
+  See below example of creating a service principal.
+
+  ![Screenshot of Creating Service Principal](./02.png)
```

Comment:
Please align the service principle creation with the rest of the Jumpstart scenarios. See [example](https://github.com/microsoft/azure_arc/blob/main/docs/azure_arc_jumpstart/azure_arc_k8s/cluster_api/capi_azure/_index.md#prerequisites)

Reply:
Changes done.